src/api/firebase.js

```diff
@@ -1,7 +1,24 @@
-import { collection, onSnapshot, doc, setDoc } from 'firebase/firestore';
+import {
+	collection,
+	onSnapshot,
+	doc,
+	setDoc,
+	getCountFromServer,
+} from 'firebase/firestore';
 import { db } from './config';
 import { getFutureDate } from '../utils';
 
+/**
+ * Return whether a collection has documents in it (list token exists) or is empty (list token does not exist)
+ * @param {string} listId the listId that a user submitted to join a list
+ * @returns {boolean}
+ */
+export async function doesCollectionExist(listId) {
```

Comment:
Nice work building this out!

Reply:
Thanks! Took us a while to figure out how to get Firebase to let us know if a list exists or not! @Kmachappy was a champ on this one, and kept us both motivated to keep digging until we found a clean answer.

Reply:
Brilliant! Nice work, team!

Reply:
@a-fink All credits to you for finding the right function, we both had a clear idea what direction to go.